src/backend/distributed/sql/datatypes/citus_cluster_clock/11.2-1.sql

```diff
@@ -130,4 +130,4 @@ CREATE SEQUENCE citus.pg_dist_clock_logical_seq START 1;
 
 ALTER SEQUENCE citus.pg_dist_clock_logical_seq SET SCHEMA pg_catalog;
 
-REVOKE UPDATE ON SEQUENCE pg_catalog.pg_dist_clock_logical_seq FROM public;
+GRANT UPDATE ON SEQUENCE pg_catalog.pg_dist_clock_logical_seq to public;
```

Comment:
Do you see a problem with a non-privileged  user doing: `SELECT setval('pg_dist_clock_logical_seq', 50)`?  Can that break anything?


In general, we seem to be using `GetUserIdAndSecContext ` trick to allow non-super users to change read/sequence in the C code, see for example: `GetNextGroupId()`

I think we might consider doing that?

Reply:
>>Do you see a problem with a non-privileged user doing: SELECT setval('pg_dist_clock_logical_seq', 50)? Can that break anything?

Yes, it will break the functionality, for that exact reason I explicitly revoked it.

Reply:
Revoked all permissions as per discussion.